File: config/paths_local.example.m
function p = paths_local(p)
%PATHS_LOCAL  Per-machine path overrides (MATLAB) -- EXAMPLE TEMPLATE.
%
%   Copy this file to config/paths_local.m (same folder) and edit. paths_local.m
%   is gitignored, so each machine can point at local data / toolboxes without
%   touching tracked code. lab_paths() calls this LAST, so anything you set here
%   wins. Only override what differs on this machine.
%
%   The function receives the fully-derived struct `p` and must return it.

    % Example: data on a fast local drive on the acquisition PC.
    % p.data_root = 'D:\Imaging_Data';

    % Example: prepend a machine-local toolbox copy so it is found first.
    % p.toolboxes = [ {'C:\Codes'}; p.toolboxes ];
end

File: config/paths_local.example.py
"""
Per-machine path overrides (Python) -- EXAMPLE TEMPLATE.
================================================================================

Copy this file to `paths_local.py` (same folder) and edit. `paths_local.py` is
gitignored, so each machine can point at local data without touching tracked
code. paths_config.py imports it LAST, so anything you set here wins.

Only redefine what differs on this machine; delete the rest.
"""

from pathlib import Path

# Example: data lives on a fast local drive on the acquisition PC.
# DATA_ROOT = Path(r"D:\Imaging_Data")

# Example: write figures to a local scratch folder instead of the network share.
# FIGURES_ROOT = Path(r"C:\fiber_outputs\Figures")
# SPECTRAL_OUTPUT_ROOT = FIGURES_ROOT / "Spectral_data_outputs"
